api/models/request.go

```diff
@@ -15,6 +16,7 @@ type Req struct {
 	MaxPoints    uint32                     `json:"maxPoints"`
 	RawInterval  uint32                     `json:"rawInterval"` // the interval of the raw metric before any consolidation
 	Consolidator consolidation.Consolidator `json:"consolidator"`
+	Node         *cluster.Node              `json:"node"`
```

Comment:
we don't really need to encode the Node into json. the only place we decode incoming Req json is /cluster/getdata which uses getTargetsLocal which doesn't care about this attribute.

I guess it could be useful maybe to see the node name whin sniffing traffic for troubleshooting (though you could also look at the dest ip), but then again all Node attributes are private so it would just be `{}` in json.

we should probably use `json:"-"` here